### ISSUE TITLE
Calling Accounts().Delete() doesnt remove files from disk

### ISSUE THREAD

**Issue by ripienaar** [synadia-io/jwt-auth-builder.go#22]
After `operator.Accounts().Delete(account.Name())` and `auth.Commit()` files are still on disk

**Comment by ripienaar** [comment]
@aricart curious of your thoughts here, we're working on terraform support for accounts and delete is pretty key there, not sure what's the right thing like if we delete a operator should all files really be gone etc?

Eitherway a delete feature would be good - one that auto revoke what needs revoking etc. 

/cc @ploubser

**Comment by aricart** [comment]
Yes I will do this - need a way to backup/restore in nsc before this gets implemented.

**Comment by Green7** [comment]
Hi, I’m running into the same issue
It looks like `func (a *NscProvider) Store(operators []*authb.OperatorData)` doesn’t handle account deletions, so calling `Accounts().Delete()` doesn’t actually remove files.

@aricart any rough idea when a fix might be expected? If helpful, I could also try putting together a PR.
Thanks!

**Comment by aricart** [comment]
I had merged some of the support on the other side. But the logic is mostly nsc only - flushed some of the issues here.

**Comment by Green7** [comment]
@aricart Great! I'll see if this solves my problem.
